ui/app/adapters/mfa-method.js

```diff
@@ -0,0 +1,31 @@
+import ApplicationAdapter from './application';
+
+export default class MfaMethodAdapter extends ApplicationAdapter {
+  namespace = 'v1';
+
+  urlForQuery(methodType) {
+    let baseUrl = this.buildURL() + '/identity/mfa/method';
```

Comment:
Ember Data uses the model name among other things when building the url so the path here would likely end up being `v1/mfa-methods/identity/mfa/method`. If you override `pathForType` to return `identity/mfa/method` then you could update this to `const baseUrl = this.buildURL();` and append the method type.

Here's some more information on customizing adapters
https://guides.emberjs.com/v3.28.0/models/customizing-adapters/#toc_path-customization

Reply:
I tried using pathForType and buildUrl but for whatever reason it doesn't modify the path. We can revisit this later.